stdlib/public/SwiftShims/RefCount.h

```diff
@@ -407,17 +415,16 @@ class RefCountBitsT {
   constexpr
   RefCountBitsT(uint32_t strongExtraCount, uint32_t unownedCount)
     : bits((BitsType(strongExtraCount) << Offsets::StrongExtraRefCountShift) |
-           (BitsType(1)                << Offsets::IsImmortalOrFastDeallocShift) |
+           (BitsType(1)                << Offsets::FastDeallocShift) |
            (BitsType(unownedCount)     << Offsets::UnownedRefCountShift))
   { }
   
   LLVM_ATTRIBUTE_ALWAYS_INLINE
   constexpr
   RefCountBitsT(Immortal_t immortal)
-    : bits((BitsType(2) << Offsets::StrongExtraRefCountShift) |
-           (BitsType(2) << Offsets::UnownedRefCountShift) |
-           (BitsType(1) << Offsets::IsImmortalOrFastDeallocShift) |
-           (BitsType(1) << Offsets::UseSlowRCShift))
+  : bits((BitsType(2) << Offsets::StrongExtraRefCountShift) |
+         (BitsType(~0 & Offsets::IsImmortalBitCount) << Offsets::IsImmortalShift) |
```

Comment:
I don't think this does what you want it to do…

(Also, boo.)

Reply:
lol. Yeah, I need to sit down and actually work through this rather than just throwing things at CI.

Reply:
Oh, yeah, there was supposed to be a shift in there somewhere whoops.